# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../iproute2/0001-include-sys-param.h.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

include sys/param.h to get MIN declared

------------------------------------------------------------------------------
tc_stab.c: In function 'print_size_table':
tc_stab.c:129:5: error: implicit declaration of function 'MIN' [-Werror=implicit-function-declaration]
------------------------------------------------------------------------------

detected when build in a musl libc based environment

--- ./tc/tc_stab.c.orig	2013-08-29 01:52:26.979474774 +0200
+++ ./tc/tc_stab.c	2013-08-29 01:56:33.381181386 +0200
@@ -21,6 +21,7 @@
 #include <arpa/inet.h>
 #include <string.h>
 #include <malloc.h>
+#include <sys/param.h>
 
 #include "utils.h"
 #include "tc_util.h"
--- ./tc/tc_util.c.orig	2013-08-29 01:55:06.383168044 +0200
+++ ./tc/tc_util.c	2013-08-29 01:55:55.476313128 +0200
@@ -20,6 +20,7 @@
 #include <arpa/inet.h>
 #include <string.h>
 #include <math.h>
+#include <sys/param.h>
 
 #include "utils.h"
 #include "tc_util.h"
